### DISCUSSION TITLE
Using  AdNauseam on Youtube videos and it says "No ads recognised in this page", is this ok?

### DISCUSSION THREAD

**Discussion by B-to480** [dhowe/AdNauseam#2060] [Q&A] [answered]
I'm asking if, while watching a YT video, this extention is working properly or not. When I'm on the main page of Youtube, it is working, the extention shows a few ads blocked and clicked, but when I watch a video, it doesn't show ads blocked or updates the list.
Also, I want to know if this extention works to support a YT channel with clicking the ads for me when watching a video, or do I have to turn off the extention on YT?
Thanks for reading!

**Comment by dhowe** [comment, marked as answer]
Thanks for the question - AdNauseam currently blocks ads on youtube video pages, but does not collect or click them. This is due to the fact that we have not as yet found a reliable method for collecting interstitial video ads, so in the meantime we block them outright (as uBlock Origin or other adblockers would do). But if you want to support a particular YT channel, you can alwyas add that page to AdNauseam's whitelist, in Settings.

**Reply by ajamico** [reply]
How would one whitelist a certain youtube channel like you mentioned here? Or were you suggesting to whitelist each specific video? Sorry if I misunderstood

**Reply by B-to480** [reply]
Hi! I think They mean add Youtube to the whitelist, @ajamico you sure have noticed that the extention can't do more than that. Let's hope that they can add more interesting stuff to it and keep it safe and functional :) . Have a good day!

**Reply by mneunomne** [reply]
Hello @ajamico. One can whitelist a specific video but currently it is not possible a specific channel. This is due to the fact that youtube doesn't have the channel name or id in the video path, only the video id.

You can easily whitelist a specific page by using the new UI on the "disable" button on the menu, by selecting "ON THIS PAGE" on the popup menu:

<img width="322" alt="Screenshot 2022-05-21 at 18 34 34" src="https://user-images.githubusercontent.com/4967860/169660948-1a4c3a3f-5775-4e6a-bda1-62f7dda41e3f.png">

Thank you for sharing some feedback :)

**Reply by studiofruits** [reply]
thank you for this, I will use this instead of skipvids.com now
